include/mme-app/mme_app.h

```diff
@@ -12,6 +12,20 @@
 #include <stdbool.h>
 
 #include "s1ap_structs.h"
+
+typedef struct dns_config
+{
+        unsigned int dns_flag;
```

Comment:
This structure should have dns_enabled flag. With default value = False.

Reply:
Noted